lib/internal/util/inspect.js

```diff
@@ -1250,7 +1250,7 @@ function getFunctionBase(value, constructor, tag) {
   if (value.name === '') {
     base += ' (anonymous)';
   } else {
-    base += `: ${value.name}`;
+    base += `: ${typeof value.name === 'symbol' ? inspect(value.name) : value.name}`;
```

Comment:
```suggestion
    base += `: ${typeof value.name !== 'string' ? formatValue(ctx, (value.name) : value.name}`;
```

Calling inspect inside of inspect is going to have new indentations, so we should instead use the internal method. I think it's also a good idea to format everything in an identical way, so we can just use it for all types besides strings.